packages/ui/src/components/button-with-options.tsx

```diff
@@ -71,25 +61,18 @@ export const ButtonWithOptions = <T extends string>({
   handleOptionChange,
   className,
   buttonClassName,
-  size = 'default',
```

Comment:
`ButtonWithOptions` will not have size prop, it will be always default size.

Reply:
what is a `ButtonWithOptions` component? have we added docs for it?
I don't see it tracked in our sheet as a separate component.